server/src/main/java/org/elasticsearch/health/HealthPeriodicLogger.java

```diff
@@ -212,6 +212,18 @@ static Map<String, Object> convertToLoggedFields(List<HealthIndicatorResult> ind
             );
         });
 
+        // message field. Show the non-green indicators if they exist.
+        List<String> nonGreen = indicatorResults.stream()
+            .filter(p -> p.status().xContentValue().equals("green") == false)
```

Comment:
I think this could be
```suggestion
            .filter(p -> p.status() != HealthStatus.GREEN)
```